watertap/data/techno_economic/component_list.yaml

```diff
@@ -117,3 +117,5 @@ viruses_enteric:
   constituent_longform: Viruses (enteric)
 zinc:
   constituent_longform: Zinc
+CO2:
```

Comment:
Minor comment- to be more in line with how we named other components, perhaps this should be renamed `carbon_dioxide`.

Reply:
Yes, I agreed this and revised to `carbon_dioxide` accordingly.